contracts/voting/cwd-voting-cw4/src/state.rs

```diff
@@ -16,4 +16,4 @@ pub const TOTAL_WEIGHT: SnapshotItem<Uint128> = SnapshotItem::new(
 );
 
 pub const GROUP_CONTRACT: Item<Addr> = Item::new("group_contract");
-pub const DAO_ADDRESS: Item<Addr> = Item::new("dao_address");
+pub const DAO: Item<Addr> = Item::new("dao");
```

Comment:
we actually need to leave the key the same here. otherwise, when we upgrade this contract the query will stop working (it'll look up the wrong key). could either add a migrate method to this, or add something along these lines to a comment and keep the key as `dao_address`.

```suggestion
pub const DAO: Item<Addr> = Item::new("dao_address");
```

Reply:
HOLY SHIT you are right... this would not have been fun.